ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -69,12 +69,34 @@ UCR_TIMING_FILE = os.path.join(LOG_HOME,"{{ localsettings.DEPLOY_MACHINE_NAME }}
 UCR_DIFF_FILE = os.path.join(LOG_HOME,"{{ localsettings.DEPLOY_MACHINE_NAME }}-ucr_diff.log")
 UCR_EXCEPTION_FILE = os.path.join(LOG_HOME,"{{ localsettings.DEPLOY_MACHINE_NAME }}-ucr_exception.log")
 
+# Shared Drive Setup
+{% if shared_drive_enabled|default(true) %}
+SHARED_DRIVE_ROOT = '{{ shared_mount_dir }}'
+{% else %}
+SHARED_DRIVE_ROOT = '{{ shared_data_dir }}'
+{% endif %}
+RESTORE_PAYLOAD_DIR_NAME = '{{ restore_payload_dir_name }}'
+TRANSFER_FILE_DIR_NAME = '{{ transfer_payload_dir_name }}'
+SHARED_TEMP_DIR_NAME = '{{ shared_temp_dir_name }}'
+TRANSFER_SERVER = '{{ (proxy_type == 'nginx') | ternary('nginx', 'apache') }}'
+
+
 # Email setup
 # email settings: these ones are the custom hq ones
+{% if file_email or not localsettings.EMAIL_LOGIN %}
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
+EMAIL_FILE_PATH = '/tmp/django_email'
```

Comment:
i think its nice to have if we ever set up a cluster again that does not have access to an email backend at the time of setup, so that we can proceed with the setup and use file emails until we get smtp. and it still works normally for the default case.